ddtrace/tracer/option_test.go

```diff
@@ -129,4 +129,38 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		assert.Equal("testEnv", c.globalTags[ext.Environment])
 		assert.True(c.debug)
 	})
+
+	t.Run("envs", func(t *testing.T) {
+		if err := os.Setenv("DD_SERVICE_NAME", "TEST_SERVICE"); err != nil {
+			panic("could not set environment variable DD_SERVICE_NAME during testing")
+		}
+		if err := os.Setenv("DD_TAGS", "env:test, aKey:aVal,bKey:bVal"); err != nil {
+			panic("could not set environment variable DD_TRACE_GLOBAL_TAGS during testing")
+		}
+
+		assert := assert.New(t)
+		var c config
+		defaults(&c)
+		assert.Equal("TEST_SERVICE", c.serviceName)
+
+		env, ok := c.globalTags["env"]
+		assert.True(ok, "has the env key")
+		assert.Equal("test", env)
+
+		aVal, ok := c.globalTags["aKey"]
+		assert.True(ok, "has aKey key")
+		assert.Equal("aVal", aVal)
+
+		bVal, ok := c.globalTags["bKey"]
+		assert.True(ok, "has bKey key")
```

Comment:
Can you please remove the comments from all of these? I'm not sure they help.
```suggestion
		assert.True(ok)
```